applets/clock/clock.c

```diff
@@ -51,6 +51,12 @@
 #include <gdk/gdkkeysyms.h>
 #include <gio/gio.h>
 
+#if defined CLOCK_INPROCESS && defined HAVE_WAYLAND
```

Comment:
style: as you want, but I'm used to `defined(x)` rather than `defined x` and you use the former below

Reply:
Well, they are both stated to be valid code

Reply:
Those have been changed to use the parentheses